helm/opendistro-es/files/action_file.yml

```diff
@@ -108,3 +108,38 @@ actions:
       unit_count: {{ .Values.curator.retention.kubernetes_days }}
     - filtertype: kibana
       exclude: True
+  7:
```

Comment:
Can you update the numbers here to 9 an 10?
https://github.com/elastisys/opendistro-build/blob/main/helm/opendistro-es/files/postgresql-actions.yml

Reply:
sure